projects/example-project-webpack/webpack.config.js

```diff
@@ -126,6 +126,20 @@ const shared = (env) => {
             host: '0.0.0.0',
             disableHostCheck: true,
         },
+        optimization: {
+            splitChunks: {
+                cacheGroups: {
+                    vendor: {
+                        // Work around to "trick" webpack to see local packages as "node_modules".
+                        test: /[\\/](node_modules|packages)[\\/]/,
+                        name: 'vendor',
+                        // "async" will ensure the dynamic imports mark the boundary between chunks
+                        // "all" will allow webpack to make decisions about chunk breaking for itself
+                        chunks: 'async',
```

Comment:
Managing this directly is something that _maybe_ applications should be doing anyways? `all` ensures you're using a single "external" bundle or vendor code, and then a small bundle for your application code:
![image](https://user-images.githubusercontent.com/1156657/89314748-0cdcae80-d648-11ea-9275-a3c4d0d372d2.png)
![image](https://user-images.githubusercontent.com/1156657/89314799-1b2aca80-d648-11ea-9411-689e7ce9b3b8.png)
Kind on interesting as it sequesters "low change rate" vendor code to its own bundle _AND_ chooses NOT to use the dynamic import boundary to define chunks.